update_existing_employee_webhook_receiver.py

```diff
@@ -16,32 +17,45 @@ def update_employee_in_guide(event, context):
     try:
         # Fetch the Builder API key, the guide ID of the guide where the content
         # is published, and the custom list ID that the items are associated with
-        api_key, guide_id, employee_customlist_id, zenefits_app_key = fetch_ssm_params()
+        api_key, guide_and_list_ids, zenefits_app_key = fetch_ssm_params()
 
         # Initialize a Session
         session = requests.Session()
         session.headers.update({"Authorization": f"JWT {api_key}"})
 
-        # Use the lambda event data to build a CustomListItem
-        data = event["data"]
-        customlist_data_builder = CustomlistDataBuilder(guide_id, zenefits_app_key)
-        customlist_data = customlist_data_builder.build(data)
+        for guide_id, employee_customlist_id in guide_and_list_ids:
+            # Use the lambda event data to build a CustomListItem
+            data = event["data"]
+            customlist_data_builder = CustomlistDataBuilder(guide_id, zenefits_app_key)
+            customlist_data = customlist_data_builder.build(data)
 
-        # Fetch the existing CustomListItem from Builder by filtering on the import_id field.
-        # This is needed to obtain the CustomListItem.id, which is required in the PATCH request
-        url = "https://builder.guidebook.com/open-api/v1/custom-list-items?guide={}&custom_lists={}&import_id={}".format(
-            guide_id, employee_customlist_id, data["id"]
-        )
-        response = session.get(url)
-        response.raise_for_status()
-        custom_list_item = response.json()["results"][0]
+            # Fetch the existing CustomListItem from Builder by filtering on the import_id field.
+            # This is needed to obtain the CustomListItem.id, which is required in the PATCH request
+            url = "https://builder.guidebook.com/open-api/v1/custom-list-items?guide={}&custom_lists={}&import_id={}".format(
+                guide_id, employee_customlist_id, data["id"]
+            )
+            response = session.get(url)
+            response.raise_for_status()
+            custom_list_item = response.json()["results"][0]
 
-        # Update the existing CustomListItem
-        url = "https://builder.guidebook.com/open-api/v1/custom-list-items/{}/".format(
-            custom_list_item["id"]
-        )
-        patch_response = session.patch(url, data=customlist_data)
-        patch_response.raise_for_status()
+            # Update the existing CustomListItem
+            url = "https://builder.guidebook.com/open-api/v1/custom-list-items/{}/".format(
+                custom_list_item["id"]
+            )
+            photo_available = False
+            if employee_data.get('photo_url'):
+                img_response = requests.get(employee_data['photo_url'])
+                photo_available = True if img_response.status_code == 200 else False
+
+            if photo_available:
+                with open('/tmp/image.jpg', 'wb') as handler:
```

Comment:
would it make sense to break out this `'/tmp/image.jpg'` into its own constant? Even though it's unlikely to change it seems like it's being referenced frequently